tensorflow/python/ops/image_ops_impl.py

```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+    ```python
+    >>> x = tf.random.normal(shape=(256, 256, 3))
```

Comment:
Don't use a random image; instead use a small image (like a 1x2) and show the result so this can be a doctest

Reply:
Please can I use an image from a url?

Reply:
I prefer a small image and showing the result as it doesn't require internet access on the testing harness

Reply:
I have made the changes please check to see if its okay.